### ISSUE TITLE
Is this project no longer being worked on?

### ISSUE THREAD

**Issue by GOvEy1nw** [pluja/whishper#122]
Hey, just wondering if this project is still planned to be worked on or if you've decided to move onto something else?

**Comment by pluja** [comment]
Yes! I do have plans to continue on this. However, currently I'm unable to find enough time for everything, so this project is stalled. I do want to release the [new version](https://github.com/pluja/whishper/tree/v4), and continue working on it further. I can't give a date on when I will be able to work on it again, but I hope it is soon.

**Comment by IanWardell** [comment]
@pluja  just fyi I tested this on Windows 11 with WSL2 - ubuntu and docker desktop. I was able to run it with GPU, RTX 3090 no issues from the GPU Docker Compose you provided. 

I know the docs here and with whisper (https://whishper.net/guides/gpu/) say windows gpu is untested - but I'm seeing it work.

**Comment by pluja** [comment]
> @pluja  just fyi I tested this on Windows 11 with WSL2 - ubuntu and docker desktop. I was able to run it with GPU, RTX 3090 no issues from the GPU Docker Compose you provided. 
> 
> I know the docs here and with whisper (https://whishper.net/guides/gpu/) say windows gpu is untested - but I'm seeing it work. 

Awesome, thanks for the update on this

**Comment by BrokenSword808** [comment]
> [@pluja](https://github.com/pluja) just fyi I tested this on Windows 11 with WSL2 - ubuntu and docker desktop. I was able to run it with GPU, RTX 3090 no issues from the GPU Docker Compose you provided.
> 
> I know the docs here and with whisper (https://whishper.net/guides/gpu/) say windows gpu is untested - but I'm seeing it work.

@IanWardell How did you set it up? I've replaced the .yml file with the gpu internals but I don't know where to run docker-compose up -d, using docker on windows 11.

**Comment by DevDema** [comment]
I hope the original author does not hate too much for this (I would love to open a contribution on his repository if he likes it).

I'm actively working on a fork of this project, called whishper-reloaded. Of course, I gave pluja the proper credit they deserve!
https://github.com/DevDema/whishper-reloaded
